packages/dropdown-react/src/Dropdown.tsx

```diff
@@ -5,9 +5,14 @@ import nanoid from "nanoid";
 import { SupportLabel } from "@fremtind/jkl-typography-react";
 import { useListNavigation } from "./useListNavigation";
 
+export type SelectValue = {
```

Comment:
Should this be exported from `index.ts` as well?

Reply:
Not a huge fan of the name. SelectValuePair or SelectObject maybe